cluster-management/setup/cluster-discovery/index.md

```diff
@@ -28,7 +28,7 @@ Boot each one of the machines with identical cloud-config and they should be aut
 #cloud-config
 
 coreos:
-  etcd:
+  etcd2:
```

Comment:
sorry for the delay... (I was debugging etcd)

We still need to change line 39 to etcd2.service and line 36-37 to use new flags.